ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/UserInterface/01e_Search.md

# Search

*Accounting > Post > Tab SEARCH*

![Search](../../Assets/Screenshots/RetailSuiteAccounting/Book/Search/Search01.png "[Search]")

> The *SEARCH* tab in the *Accounting* module allows to search for a specific account or posting using different search criteria. For detailed information about the search function, see [Search in accounting](../Operation/14_SearchAccounting.md).

Reviewer: ```suggestion
When clicking the *SEARCH* tab, the search side bar is displayed. You can search for a specific account or posting using different search criteria. For detailed information about searching for an account or a posting, see [Search in accounting](../Operation/14_SearchAccounting.md).
```

> - ![Left arrow](../../Assets/Icons/Back02.png "[Left arrow]") (Left arrow)   
> Click this arrow to open the *Post extras* side bar.
> 
>   > [Info] The *Post extras* side bar is opened automatically when clicking the *SEARCH* tab.
> 
> - ![Right arrow](../../Assets/Icons/Close.png "[Right arrow]") (Right arrow)  
> Click this arrow to close the *Post extras* side bar.

Reviewer: ```suggestion
- ![Right arrow](../../Assets/Icons/Close.png "[Right arrow]") (Right arrow)  
Click this arrow to fold in the *Post extras* search side bar.

- ![Left arrow](../../Assets/Icons/Back02.png "[Left arrow]") (Left arrow)   
Click this arrow to fold out the *Post extras* search side bar.
```

> The *Post extras* side bar contains two sub-tabs:
> 
>   - *Accounts*: to search for accounts
>   - *Search*: to search for postings

Reviewer: ```suggestion
The *Post extras* search side bar contains two tabs:
- *Accounts*: Search for accounts
- *Search*: Search for postings
```

> ## Accounts sub-tab

Reviewer: ```suggestion
## Accounts
```

> *Accounting > Post > Tab SEARCH > Side bar Post extras > Sub-tab Accounts*

Reviewer: ```suggestion
*Accounting > Post > Tab SEARCH > Tab Accounts*   
*Accounting > Post > Unfold search side bar > Tab Accounts*
```

>   ![Sub-tab Accounts](../../Assets/Screenshots/RetailSuiteAccounting/Book/Search/BookExtrasAccounts.png "[Sub-tab Accounts]")
> 
>   ![Search for an account](../../Assets/Screenshots/RetailSuiteAccounting/Book/Search/AccNoDescription.png "[Search for an account]")
> 
>   [comment]: <> (Erstes Bild muss ersetzt werden - ohne Wellen bzw. ohne zu kürzen. Oder alternativ, zweites Bild? RS mit HG notwendig.)

Reviewer: ```suggestion
  ![Sub-tab Accounts](../../Assets/Screenshots/RetailSuiteAccounting/Book/Search/BookExtrasAccounts.png "[Sub-tab Accounts]")
```
Bild 1  wie besprochen kürzen Bild 2 raus

- [QUICK SEARCH]  

> Click this button to display the *QUICK SEARCH* context menu. The following options are available:
> 
>   - ![Checkbox](../../Assets/Icons/Checkbox.png "[Checkbox]") *Acc. no.*  
>   Select or deselect this checkbox to include or exclude this criteria in the search. This checkbox is selected by default.
> 
>   - ![Checkbox](../../Assets/Icons/Checkbox.png "[Checkbox]") *Description*  
>   Select or deselect this checkbox to include or exclude this criteria in the search. This checkbox is selected by default.

Reviewer: ```suggestion
  Click this button to display the *QUICK SEARCH* filter list. The following entries are displayed:
  - ![Checkbox](../../Assets/Icons/Checkbox.png "[Checkbox]") *Acc. no.*  
    Select this checkbox to use the account number as a search criterion. By default, this checkbox is selected.
  - ![Checkbox](../../Assets/Icons/Checkbox.png "[Checkbox]") *Description*  
    Select this checkbox to use the account description as a search criterion. By default, this checkbox is selected.
```

- [Search bar]  

> Enter the search text (letters or numbers).

Reviewer: ```suggestion
Enter the search text (letters or numbers). The criteria selected in the *QUICK SEARCH* filter list is applied to the search.
```

- ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross)  

> Click this button to delete the search text and to clear the search results.

Reviewer: ```suggestion
Click this button to clear the search bar.
```

- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)  

> Click this button to start the search. The search can also be started by pressing ENTER.

Reviewer: ```suggestion
Click this button to start the search. Alternatively, you can press the **ENTER** key.
```

The available accounts are displayed in a column view. This information is read-only. The accounts are set up during the configuration process. For detailed information, see [Chart of accounts](../Integration/01_RunAccountingWizard.md#chart-of-accounts).

- *Acc. no.*  
This column displays the account number.

- *Description*  
This column displays the account name.

[comment]: <> (Info unten weglassen? Standardfunktionen)

> - ![First page](../../Assets/Icons/FirstPage.png "[First page]") - ![Last page](../../Assets/Icons/LastPage.png "[Last page]") (First page) - (Last page)  
>  ![Previous page](../../Assets/Icons/PreviousPage.png "[Previous page]") - ![Next page](../../Assets/Icons/NextPage.png "[Next page]") (Previous page) - (Next page)  
> Click these buttons to navigate the accounts list.
> 
> - *Page [x] of [x]*  
> Enter a page number to go directly to a specific page.
> 
> - ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
>  Click this button to update the accounts list after making changes in the accounts. For detailed information about creating, editing or deleting accounts, see [Manage the accounts](../Integration/02_ManageAccounts.md).

Reviewer: ```suggestion
- ![First page](../../Assets/Icons/FirstPage.png "[First page]") (First page)    
  Click this button to display the first page of the search results.
- ![Last page](../../Assets/Icons/LastPage.png "[Last page]") (Last page)    
 Click this button to display the last page of the search results.
- ![Previous page](../../Assets/Icons/PreviousPage.png "[Previous page]") (Previous page)   
   Click this button to display the previous page of the search results.
- ![Next page](../../Assets/Icons/NextPage.png "[Next page]") (Next page)     
  Click this button to display the next page of the search results.

- *Page*     
  Enter a page number to go to the entered page.

- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
 Click this button to update the search results.
```

> ## Search sub-tab
> 
> *Accounting > Post > Tab SEARCH > Side bar Post extras > Sub-tab Search*
> 
>   ![Sub-tab Search](../../Assets/Screenshots/RetailSuiteAccounting/Book/Search/BookExtrasSearch.png "[Sub-tab Search]")

Reviewer: ```suggestion
## Search

*Accounting > Post > Tab SEARCH > Tab Search*

![Sub-tab Search](../../Assets/Screenshots/RetailSuiteAccounting/Book/Search/BookExtrasSearch.png "[Sub-tab Search]")
```

> - *Active* [checkbox]  
> Select o deselect this checkbox next to a search criterion to activate or deactivate the corresponding search criterion. All checkboxes are deselected by default.

Reviewer: ```suggestion
- [x] *Active*    
    Select the checkbox to activate the corresponding search criteria. By default, no search criterion is selected. 
```

- *Search criterion*  

> This column displays a list of the available search criteria. The following search criteria are available:

Reviewer: ```suggestion
  The following search criteria are available:
```

  - *Turnover*  
  Enter the turnover value to be searched for.

  - *Contra account*  
  Enter a contra account number to be searched for.

  - *Account*  
  Enter a account number to be searched for.

  - *Receipt 1*  
  Enter a receipt 1 value to be searched for.

  - *Receipt 2*  
  Enter a receipt 2 value to be searched for.

  - *Text*  
  Enter a (posting) text to be searched for.

>   - *Fiscal month*  
>     Click the drop-down list to select the appropriate month. The following months are available:
> 
>       - **0 - Balances carried forward**
>       - **1 - 12 - (calendar month)**
>       - **13 - 15 - Subsequent postings**
>       - **16 - 25 - Closing postings**
>       - **99 - General month**  

Reviewer: ```suggestion
  - *Fiscal month*  
    Click the drop-down list to select the appropriate month. The following months are available:
      - **0 - Balances carried forward**
      - **1 - 12 - (calendar month)**
      - **13 - 15 - Subsequent postings**
      - **16 - 25 - Closing postings**
      - **99 - General month**  
```

  - *Journal*  
  Enter the journal to be searched for.

  - *Date*  

>   Enter the date to be searched for in the fields provided. The valid date format is DD MM YYYY in this case.

Reviewer: ```suggestion
  Enter the date to be searched for in the fields provided. Enter the day in the first field, the month in the second field and the year in the third field. The valid date format is DD MM YYYY.
```

  - *Cost center*  
  Enter the cost center to be searched for.

  - *Cost unit*  
  Enter the cost unit to be searched for.

  - *Processed*  
  Select the *Posting processed* checkbox to search for postings with this status.

  - *User*  

>   Click the drop-down list to select the relevant user.

Reviewer: ```suggestion
  Click the drop-down list to select the user. All users registered in the account are displayed in the list.
```

> - NOT  

Reviewer: ```suggestion
- *NOT*     
```

> Select this checkbox to exclude the entered value.

Reviewer: ```suggestion
Select this checkbox to exclude the entered value. This checkbox is displayed for the following search criteria:
- *Turnover*
- *Receipt 1*
- *Receipt 2*
- *Text*
- *Fiscal month*
- *Journal*
- *Date*
- *Cost center*
- *Cost object*
- *Processed*
- *User*
```

> - Search for  

Reviewer: ```suggestion
- *Search for*  
```

> Enter the search text in this search bar or select the appropriate option from the corresponding drop-down list.  

Reviewer: ```suggestion
Enter the search value for the corresponding search criterion in the field or select the appropriate option from the corresponding drop-down list.  
```

>   > [Info] There are two wildcards available:  
>   - an asterisk (*) represents any number of characters
>   - a question mark (?) represents a single character

Reviewer: ```suggestion
  > [Info] You can enter the following wildcards in the *Search for* fields:  
  - an asterisk (*) represents any number of characters
  - a question mark (?) represents a single character
```

- [SEARCH]  

> Click this button to start the search. The search can also be started by pressing ENTER.

Reviewer: ```suggestion
Click this button to start the search. Alternatively, you can press the **ENTER** key. All postings matching the search criteria are displayed in the *SEARCH* tab.
```

  - *Account*  
  Enter a account number to be searched for.

  - *Receipt 1*  
  Enter a receipt 1 value to be searched for.

  - *Receipt 2*  
  Enter a receipt 2 value to be searched for.

  - *Text*  
  Enter a (posting) text to be searched for.

  - *Fiscal month*  
    Click the drop-down list to select the appropriate month. The following months are available:
      - **0 - Balances carried forward**
      - **1 - 12 - (calendar month)**
      - **13 - 15 - Subsequent postings**
      - **16 - 25 - Closing postings**
      - **99 - General month**  

  - *Journal*  
  Enter the journal to be searched for.

  - *Date*  
  Enter the date to be searched for in the fields provided. Enter the day in the first field, the month in the second field and the year in the third field. The valid date format is DD MM YYYY.

  - *Cost center*  
  Enter the cost center to be searched for.

  - *Cost unit*  
  Enter the cost unit to be searched for.

  - *Processed*  
  Select the *Posting processed* checkbox to search for postings with this status.

  - *User*  
  Click the drop-down list to select the user. All users registered in the account are displayed in the list.

- *NOT*      

>   Select this checkbox to exclude the entered value. Select this checkbox to exclude the entered value. This checkbox is displayed for the following search criteria:

Reviewer 2: doppelt
